packages/core/src/components/badge/badge.scss

```diff
@@ -0,0 +1,28 @@
+:host {
+  display: inline-block;
+}
+
+.atom-badge {
+  border-radius: var(--spacing-base);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏If I'm not mistaken recently @schirrel make the border-radius tokens, I believe we should talk to UX to see if it will be a new token.

Reply:
Nice! I sent a message to designers in the slack thread!

Reply:
@igorwessel thats rights, @gabrielduete i brough to atomium the missing border radius that were on figma but no here. 
the figmas - https://www.figma.com/file/rC8E8nEOrESRPoQQGt4kSx/Styleguide-2.0?type=design&node-id=4167-49328&mode=design&t=gsSDMZpa3BuuHEpQ-0
the ux changelog: https://juntossomosmais.slack.com/archives/C02AD0UAS5T/p1668199458847189

Reply:
Thansk, @schirrel !

I asked the design team if we can use the 12px token, which is the maximum we have, because in the badge figma we have 16px, but 12px seems to have served me very well!

[Thread](https://juntossomosmais.slack.com/archives/C0117K4MYSW/p1692302780007119?thread_ts=1692279567.787889&cid=C0117K4MYSW)

Reply:
I talked to the design team and we agreed to use the --border-radius-medium token

Commit: https://github.com/juntossomosmais/atomium/pull/239/commits/65f592897c9e07dd58b313f3ce1ad589d6307194